rails/app/views/portal/teachers/new.html.haml

```diff
@@ -1,5 +1,4 @@
 %h1 Teacher Signup Page
-= render :partial => 'form', 
-  :locals => { :portal_teacher => @portal_teacher,
-    :school_selector => @school_selector }
+= render :partial => 'form',
+  :locals => { :portal_teacher => @portal_teacher }
 = link_to 'Back', portal_teachers_path
```

Comment:
@scytacki Should we just get rid of this view and the related cucumber tests that are failing?